src/server/tests.rs

```diff
@@ -144,6 +144,43 @@ async fn test_did_change() {
     assert_eq!(r#"from(bucket: "bucket")"#, contents);
 }
 
+#[test]
+async fn test_did_change_multiple() {
+    let server = create_server();
+    open_file(
+        &server,
+        r#"from(bucket: "bucket") |> first()"#.to_string(),
+        None,
+    )
+    .await;
+
+    let params = lsp::DidChangeTextDocumentParams {
+        text_document: lsp::VersionedTextDocumentIdentifier {
+            uri: lsp::Url::parse("file:///home/user/file.flux")
+                .unwrap(),
+            version: -2,
```

Comment:
Just curious, why the version is `-2`?

Reply:
Ah, this is a test, so it was probably randomly generated. I copy/pasted the boilerplate from another test. We don't use the version here at all (the spec is confusing about how versions are used; there's a comment in our server state implementation to that effect).